EMS/core-bundle/src/Command/ReindexCommand.php

```diff
@@ -137,8 +137,6 @@ public function reindex(string $name, ContentType $contentType, ?string $index,
         /** @var EntityManager $em */
         $em = $this->doctrine->getManager();
 
-        $em->getConnection()->getConfiguration()->setSQLLogger(null);
```

Comment:
It was not something to avoid perf issue on rebuild/reindex?

Reply:
Yes indeed my reason was on the commit message:

We can not do this anymore, doctrine now uses middlewares.
But also if you do not want query logs, run the commands in different env.
We should then better configure the monolog handlers.

So unsetting the sql logger at runtime will no longer work, we may need beter configuration of the monolog logger